### ISSUE TITLE
[request] option to disable patch certain partitions for certain devices

### ISSUE THREAD

**Issue by onano** [chenxiaolong/DualBootPatcher#23]
Hello,
I want to request this feature if is it possible to disable patching the certain partitions for certain devices
example: 
-> There are devices with small system/cache partition where system and cache cannot hold second rom, hence making dual and multi-slots just a placeholder. 

if it is possible to add overlay to disable patching for certain partition to make it user proof?

**Comment by chenxiaolong** [comment]
Yeah, I think this is doable. I'll probably add some sort of field for specifying device-specific flags to https://github.com/chenxiaolong/DualBootPatcher/blob/master/libmbp/device.cpp.

Something like:

- FLAG_SMALL_SYSTEM_PARTITION
- FLAG_SMALL_CACHE_PARTITION
- etc.

**Comment by onano** [comment]
thanks for reply, yea it would be awsome to add specific flags
